src/ReactiveUI.Wpf/ReactiveUI.Wpf.csproj

```diff
@@ -0,0 +1,38 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net45</TargetFramework>
+    <DefineConstants>$(DefineConstants);WPF</DefineConstants>
+    <Description>WPF specific extensions to ReactiveUI</Description>
+    <PackageId>reactiveui-wpf</PackageId>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\ReactiveUI\CreatesCommandBinding.cs" Link="CreatesCommandBinding.cs" />
+    <Compile Include="..\ReactiveUI\Platforms\windows-common\ActivationForViewFetcher.cs" Link="windows-common/ActivationForViewFetcher.cs" />
```

Comment:
This shows up with a little cross in VS for me (I don't think VS really likes it!) - long term, I'd be pro splitting the implementations (maybe sharing a base class for common functionality so we don't have the mess of #ifdefs) but for now, I think these might be neater just being included in the root of the project (the link should be fairly easy to spot, and the windows-common doesn't make a huge amount of difference within the context of the WPF project)